### PR TITLE
Updates to EF Core/EF6 comparison docs

### PR DESCRIPTION
**Pull request by ajcvickers**

### CODE REVIEW

File: entity-framework/efcore-and-ef6/index.md [hunk 1]
```diff
@@ -1,77 +1,74 @@
...
-| Inheritance: Table per type (TPT)                     | Yes      |                                       |
-| Inheritance: Table per concrete class (TPC)           | Yes      |                                       |
+| Inheritance: Table per type (TPT)                     | Yes      | Planned for 5.0 ([#2266](https://github.com/dotnet/efcore/issues/2266)) |
+| Inheritance: Table per concrete class (TPC)           | Yes      | Stretch for 5.0 ([#3170](https://github.com/dotnet/efcore/issues/3170)) <sup>(1)</sup> |
 | Shadow state properties                               |          | 1.0                                   |
 | Alternate keys                                        |          | 1.0                                   |
-| Many-to-many without join entity                      | Yes      |                                       |
+| Many-to-many navigations                              | Yes      | Planned for 5.0 ([#19003](https://github.com/dotnet/efcore/issues/19003)) |
+| Many-to-many without join entity                      | Yes      | Backlogged ([#1368](https://github.com/dotnet/efcore/issues/1368)) |
```
Comment (bricelam) [inline entity-framework/efcore-and-ef6/index.md]: > ~~Backlogged~~ **On the backlog**

Backlogged sounds like we were going to do it but then decided to punt it.

File: entity-framework/efcore-and-ef6/index.md [hunk 2]
```diff
@@ -1,77 +1,74 @@
...
 # Compare EF Core & EF6
 
-Entity Framework is an object-relational mapper (O/RM) for .NET. This article compares the two versions: Entity Framework 6 and Entity Framework Core.
+## EF Core
 
-## Entity Framework 6
+Entity Framework Core ([EF Core](../core/index.md)) is a modern object-database mapper for .NET. It supports LINQ queries, change tracking, updates, and schema migrations.
 
-Entity Framework 6 (EF6) is a tried and tested data access technology. It was first released in 2008, as part of .NET Framework 3.5 SP1 and Visual Studio 2008 SP1. Starting with the 4.1 release it has shipped as the [EntityFramework](https://www.nuget.org/packages/EntityFramework/) NuGet package. EF6 runs on the .NET Framework 4.x and .NET Core from 3.0 onwards.
+EF Core works with SQL Server/SQL Azure, SQLite, Azure Cosmos, MySQL, ProstgreSQL, and many more databases through a [database provider plugin model](../core/providers/index.md).
```
Comment (roji) [inline entity-framework/efcore-and-ef6/index.md]:
```suggestion
EF Core works with SQL Server/SQL Azure, SQLite, Azure Cosmos, MySQL, PostgreSQL, and many more databases through a [database provider plugin model](../core/providers/index.md).
```

File: entity-framework/efcore-and-ef6/index.md [hunk 3]
```diff
@@ -1,77 +1,74 @@
...
 
-The following tables compare the features available in EF Core and EF6. It's a high-level comparison and doesn't list every feature or explain differences between the same feature in different EF versions.
+The following tables compare the features available in EF Core and EF6. This is a high-level comparison and doesn't list every feature or explain differences between the same feature in different EF versions.
 
 The EF Core column indicates the product version in which the feature first appeared.
 
 ### Creating a model
 
-| **Feature**                                           | **EF 6** | **EF Core**                           |
+| **Feature**                                           | **EF6.4**| **EF Core**                           |
```
Comment (roji) [inline entity-framework/efcore-and-ef6/index.md]: Is it a good idea to mention the minor version? If we release another version we'd have to update it, and it's not like the EF6 feature set is going to change...

File: entity-framework/efcore-and-ef6/index.md [hunk 4]
```diff
@@ -1,77 +1,74 @@
...
-| Many-to-many without join entity                      | Yes      |                                       |
+| Many-to-many navigations                              | Yes      | Planned for 5.0 ([#19003](https://github.com/dotnet/efcore/issues/19003)) |
+| Many-to-many without join entity                      | Yes      | Backlogged ([#1368](https://github.com/dotnet/efcore/issues/1368)) |
 | Key generation: Database                              | Yes      | 1.0                                   |
 | Key generation: Client                                |          | 1.0                                   |
 | Complex/owned types                                   | Yes      | 2.0                                   |
 | Spatial data                                          | Yes      | 2.2                                   |
-| Graphical visualization of model                      | Yes      |                                       |
-| Graphical model editor                                | Yes      |                                       |
+| Graphical visualization of model                      | Yes      | No support planned <sup>(2)</sup>     |
```
Comment (roji) [inline entity-framework/efcore-and-ef6/index.md]: I'd move these (and the possibly the model format/create model items) to the bottom, after "core" features such as field mapping, table splitting...

File: entity-framework/efcore-and-ef6/index.md (hunk 1 above)
Comment (roji) [reply entity-framework/efcore-and-ef6/index.md]: Or just "Backlog"

File: entity-framework/efcore-and-ef6/index.md [hunk 5]
```diff
@@ -82,79 +79,76 @@ The EF Core column indicates the product version in which the feature first appe
...
 
-## Guidance for existing EF6 applications
+EF6.4 runs on .NET Core and .NET Framework, through multi-targeting.
 
-Because of the fundamental changes in EF Core, we do not recommend moving an EF6 application to EF Core unless there is a compelling reason to make the change. If you want to move to EF Core to use new features, make sure you're aware of its limitations. For more information, see [Porting from EF6 to EF Core](porting/index.md). **The move from EF6 to EF Core is more a port than an upgrade.**
+## Guidance for new applications
+
+Use EF Core on .NET Core for all new applications unless the app needs something that is [only supported on .NET Framework](https://docs.microsoft.com/dotnet/standard/choosing-core-framework-server).
+
+## Guidance for existing EF6 applications
```
Comment (roji) [inline entity-framework/efcore-and-ef6/index.md]: Link to the [porting guide](https://docs.microsoft.com/en-us/ef/efcore-and-ef6/porting/)?

File: entity-framework/efcore-and-ef6/index.md (hunk 2 above)
Comment (ErikEJ) [inline entity-framework/efcore-and-ef6/index.md]: "Azure SQL Database" and "Azure Cosmos DB" seem to be the official short product names

File: entity-framework/efcore-and-ef6/index.md (hunk 3 above)
Comment (ajcvickers) [reply entity-framework/efcore-and-ef6/index.md]: @roji I went both ways on this. I didn't want to give the impression that all EF6 releases run on .NET Core, so I started explicitly saying EF6.4 in some places. Then I decided to be consistent.

While it's true that we would need to update this when we do a new release, this is one of the pages we _should_ update with each release anyway.

File: entity-framework/efcore-and-ef6/index.md (hunk 5 above)
Comment (ajcvickers) [reply entity-framework/efcore-and-ef6/index.md]: Good point. Although the porting guide is also something that needs updating!

File: entity-framework/efcore-and-ef6/index.md (hunk 2 above)
Comment (ajcvickers) [reply entity-framework/efcore-and-ef6/index.md]: Thanks @ErikEJ!

**Pull request merged**